api-outbound.yaml

```diff
@@ -288,14 +291,17 @@ components:
     AssetEventWithHostname:
       type: object
       required:
-        - lastScanned
         - id
         - hostname
       properties:
         lastScanned:
           type: string
           format: date-time
-          description: The last time the asset was scanned in ISO8601 format.
+          description: The time the asset was scanned in ISO8601 format. LastScanned will get removed as a part of PSD-306
```

Comment:
nit:  "PSD-306" doesn't make sense in open source... users of this project that are not us (Atlassian) don't know what this means.